package.js

```diff
@@ -6,7 +6,7 @@ Package.describe({
 });
 
 Npm.depends({
-  twilio: '1.11.0'
+  twilio: '1.11.1'
```

Comment:
@creeonix You need to define the [exact version](http://docs.meteor.com/#/full/Npm-depends). 

This is so we have dependable builds -- in case a patch version brings in an unexpected regression.